filament/backend/test/test_LoadImage.cpp

```diff
@@ -290,31 +290,35 @@ TEST_F(BackendTest, UpdateImage2D) {
 
     api.startCapture();
 
+    Cleanup cleanup(api);
+    cleanup.addPostCall([&]() { api.finish(); });
```

Comment:
I think I prefer the `api.finish();` call to be at the end of the function rather than handled through `Cleanup`.

This is "what feels better in my gut", but I think it doesn't have enough reason to justify breaking the more general rule of "don't have a non-linear control flow in tests".